ref(docs): Update migration guide to use --require/--import flag (#11970)
getsentry/sentry-javascript: docs/v8-node.md

const app = express();
```

We recommend creating a file named `instrument.js` that imports and initializes Sentry.

```js

> // In v8, we create a file that only initializes sentry.

Reviewer: ```suggestion
// In v8, create a separate file that initializes sentry.
```

> // The file is then passed to Node via --require or --import.

Reviewer: ```suggestion
// Then pass the file to Node via --require or --import.
```

### ESM Support

Instrumentation works out of the box for CommonJS (CJS) applications based on require() calls. This means that as long

> as your application is either natively in CJS, or compiled at build time to CJS (which is the case for most TypeScript
> apps, for example), everything will work without any further setup.

Reviewer: ```suggestion
as your application is either natively in CJS, or compiled at build time to CJS, everything will work without any further setup.
```

This may not hold over time and may lead people to false security.

> ECMAScript Modules (ESM) are only supported for Node v18.19.0 onwards.

Reviewer 2: ```suggestion
Sentry supports ECMAScript modules (ESM) only for Node v18.19.0 onwards.
```
